docs/api/browser-window.md

```diff
@@ -669,6 +669,8 @@ Hides the window.
 
 Returns `boolean` - Whether the window is visible to the user.
 
+For a window to be visible, it must be visible to the user in the foreground of the app, which means that it should not be minimized or occluded.
```

Comment:
Not sure about the occluded bit here - isn't that only supported on macOS? Wouldn't want to give users the wrong impression that they can expect that behavior on Windows/Linux.

Reply:
good point, you're right and i'll remove that.